feat(changelog): vpc-added-network-acl-is-now-available-in-general-avail 2026-06-03 (#6565)
scaleway/docs-content: changelog/june2026/2026-06-03-vpc-added-network-acl-is-now-available-in-general-avail.mdx

---

> title: Network ACL is now available in General Availability

Reviewer: ```suggestion
title: Network ACLs are now in General Availability
```

status: added
date: 2026-06-03
category: network
product: vpc
---

> Network ACL allows you to define fine-grained access control policies for your network resources. Network ACL is now also available in the Console.

Reviewer: ```suggestion
Network ACLs allow you to define fine-grained access control policies for your network resources, and are now available in the Scaleway console.
```
